package.json

```diff
@@ -62,14 +63,12 @@
     "cross-env": "^7.0.3",
     "del-cli": "^5.0.0",
     "eslint": "^8.44.0",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-airbnb-typescript": "^17.0.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-import-resolver-typescript": "^3.5.5",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-prettier": "^5.0.0",
     "esm": "^3.2.25",
-    "husky": "^8.0.3",
+    "husky": "^9.0.11",
```

Comment:
I figured I'd bump Husky to the latest v9 since I was touching it anyway – but the changes in this PR should work fine with v8 too